README.md

```diff
@@ -1,2 +1,14 @@
 # pimcore-sitemap-plugin
 XML sitemap generator for Pimcore. Built from the document tree.
+
```

Comment:
Somewhere here squashed between these things we should have some installation instructions too :D, something like:

```
## Installation

To install the plugin, simply require it with Composer:

'''
$ composer require byng/pimcore-sitemap-plugin
'''
```

Make the `'''` bits actual backticks though, like ```, it's meant to be code >_<